Library.Console/BooksManger.cs

```diff
@@ -0,0 +1,24 @@
+namespace Library.Console;
+
+using System.Text.Json;
+
+public static class BooksManger
+{
+    public static void Save(List<Book> books)
```

Comment:
You can have `List` in `Save(List<Book> books)` changed to something like `IList` or `ICollection` / `IEnumerable`, but I think it's ok for now